azurerm/internal/services/machinelearning/ssl_config/key.pem

```diff
@@ -0,0 +1,28 @@
+-----BEGIN PRIVATE KEY-----
```

Comment:
could we put these in a testdata directory?

Reply:
done in the second commit.